sources/packages/web/Dockerfile.dev

```diff
@@ -1,31 +1,52 @@
-# Base Image
-FROM node:16.17.0-alpine3.16
+# "build-stage", based on Node.js, to build and compile the frontend.
+FROM node:16.17.0-alpine3.16 AS build-stage
 
 LABEL maintainer="BC GOV"
 
-# Updating container path variable with node bin
-ENV PATH $PATH:/app/node_modules/.bin
-
-# Application Port
+# Application Port.
 ENV PORT 8080
 
-# Setting app as work dir
+# Setting app as work dir.
 WORKDIR /app
 
-# Copying package.json and package-lock.json
+# Copying package.json and package-lock.json.
 COPY ./package*.json ./
 
-# Installing dependencies
+# Installing dependencies.
 RUN npm ci
 
-# Copying sources
+# Copying sources.
 COPY . .
 
 # Building app
-RUN npm run build
+RUN npm run build-local
+
+# Get a nginx image to have only the compiled app ready for production.
+FROM nginx:1.22 AS production-stage
```

Comment:
Minor but can we make the layer name `production-stage` aligned with the other Dockerfile. Mentioning production here and in the line 24 comment can be misleading.